### PR TITLE
Adding `--date' argument to configure time format

### PR DESCRIPTION
**Pull request by jazcarate**
### Summary
`--iso8601` was too hard for me to remember, and most often than not I want to see the time in my local timezone

### Preflight checklist
- [x] Code formatted with rustfmt
- [x] Relevant tests added
- [x] Any new documentation added

### CODE REVIEW

**Review by mike-engel — CHANGES_REQUESTED** [review]

File: src/cli_config.rs [hunk 1]
```diff
@@ -187,6 +199,25 @@ fn is_timestamp_or_duration(val: &str) -> Result<String, String> {
...
+fn time_format(arg: &str) -> Result<TimeFormat, String> {
+    match arg.to_uppercase().as_str() {
+        "UTC" => Ok(TimeFormat::UTC),
+        "LOCAL" => Ok(TimeFormat::Local),
+        _ => {
+            let mut parsed = Parsed::new();
+            match parse(&mut parsed, arg, StrftimeItems::new("%#z")) {
+                Ok(_) => match parsed.offset {
+                    Some(offset) => Ok(TimeFormat::Fixed(offset)),
+                    None => panic!("Should have been able to parse the offset"),
```
[the remark this answers is not in the document]
Comment (jazcarate) [reply src/cli_config.rs:211]: This panic should never ever happen. The format (`%#z`) is an _offset_ format, so there is *no way* `parsed.offset` is a `None`.
For example, a format like `"%H:%M:%S"` has a `None` `offset`.

The error when you input something that is not an offset is the `Err(_)` in the next line

File: src/cli_config.rs [hunk 2]
```diff
@@ -116,7 +117,18 @@ pub struct DecodeArgs {
...
+    #[clap(aliases = &["dates", "time"])]
+    #[clap(num_args = 0..=1)]
+    #[clap(require_equals = true)]
+    #[clap(value_parser = time_format)]
+    #[clap(default_value = None)]
+    #[clap(default_missing_value = "UTC")]
+    pub time_format: Option<TimeFormat>,
+
+    /// Display unix timestamps as ISO 8601 dates in UTC.
+    /// Use --date for finer control
```
[the remark this answers is not in the document]
Comment (mike-engel) [reply src/cli_config.rs:131]: The next version will be a major bump anyway, so this is a good opportunity to keep the tool simple

[the remark this answers is not in the document]
Comment (jazcarate) [reply src/cli_config.rs]: Removed 😃

**Review by mike-engel — APPROVED** [review]

**Comment by mike-engel** [comment]
Very sorry for the delay, but this is in [6.0.0](https://github.com/mike-engel/jwt-cli/releases/tag/6.0.0) now

**Pull request merged**